site/docs/pages/transaction/transaction.mdx

```diff
@@ -273,6 +273,70 @@ import { Transaction, TransactionButton, TransactionSponsor } from "@coinbase/on
 
 ::::
 
+### Using `calls` instead of `contracts`
+
+In addition to the `contracts` prop, the `<Transaction />` component also accepts a `calls` prop. This provides an alternative way to construct transactions. When using `calls`, you should not pass anything for the `contracts` prop.
+
+Using `calls` instead of `contracts` is a great way to bundle several operations into a single transaction for smart wallets. Externally Owned Accounts (EOAs) such as the Coinbase Wallet extension will execute each call one-at-a-time with a separate pop-up for each.
+
+The `calls` prop is an array of `Call` objects, where each `Call` is defined as:
+
+```typescript
+type Call = {
+  to: Hex;
+  data?: Hex;
+  value?: bigint;
+};
+```
+
+- `to`: The address you are executing the call on
+- `data`: The encoded function data to pass along in the call (optional)
+- `value`: The amount of ether to send along with the call, denominated in wei (optional)
+
+Here's an example of how to use the `calls` prop with the Transaction component:
+
+```tsx
```

Comment:
if you add twoslash it'll enable twoslash for this block, you can use // @noErrors 123 to suppress any errors you don't want to resolve for the example

Reply:
@alessey good call! now using twoslash 🫡